vite.config.ts

```diff
@@ -18,13 +18,18 @@ export default defineConfig(({ mode }) => {
 
   return {
     build: {
-      chunkSizeWarningLimit: 900,
       outDir,
       rollupOptions: {
         input: {
           main: resolve(__dirname, 'index.html'),
           _scrivito_extensions: resolve(__dirname, '_scrivito_extensions.html'),
         },
+        output: {
+          manualChunks: {
```

Comment:
Any metrics that point to having the same (or even better) performance with manual chunks?

Reply:
According to manual Lighthouse runs it's the same. Differences are all in the < 10 ms range.